i18n(de): update `editor-setup.mdx` (#6362)
withastro/docs: src/content/docs/en/editor-setup.mdx

Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. This plugin includes features like syntax highlighting, code completion, and formatting, and plans to add even more advanced features in the future. It is also available to all other [JetBrains IDEs with JavaScript support](https://www.jetbrains.com/products/#lang=js&type=ide).

> JetBrains' new [Fleet IDE](https://www.jetbrains.com/fleet/) includes language server support and will be able to use the official Astro tooling.

Reviewer: ```suggestion
JetBrains' [Fleet IDE](https://www.jetbrains.com/fleet/) includes language server support and uses the official Astro tooling.
```
Just like "upcoming" gets out of date, so does "new"! Better to not have any language like this for future-proofing! You can update the German accordingly to match this version, too.

Author: Makes sense, will change this!
